### ISSUE TITLE
Hello, I use all KITTI data for training. How can I find the model with the best performance from the trained models.

### ISSUE THREAD

**Issue by libingDY** [TRAILab/PDV#11]
Hello, I use all KITTI data for training. How can I find the model with the best performance from the trained models.

**Comment by jskhu** [comment]
You will need to take a subset of the training data as a validation set (I've seen numbers range from 5-10%) to verify your model. Ideally, the performance on the validation set is reflective of the performance of the test set, but from my experience, because of KITTI's dataset size, there is usually a large gap between validation performance and test performance.

**Comment by libingDY** [comment]
thank you
